src/dmd/cli.d

```diff
@@ -714,17 +718,13 @@ struct CLIUsage
     }
 
     /// CPU architectures supported -mcpu=id
-    static string mcpu()
-    {
-        return "
-CPU architectures supported by -mcpu=id:
-  =?             list information on all architecture choices
+    enum mcpuUsage = "CPU architectures supported by -mcpu=id:
+  =[h|help|?]    list information on all available choices
```

Comment:
Not sure whether we should be so explicit here. Imho just "help" would do fine too.